test/unittests/Microsoft.ServiceFabric.Services.Remoting.Tests/V2/ExceptionConvertors/FabricExceptionConvertorTest.cs

```diff
@@ -72,6 +72,9 @@ public class FabricExceptionConvertorTest
             new FabricPeriodicBackupNotEnabledException("FabricPeriodicBackupNotEnabledException"),
             new FabricValidationException("FabricValidationException"),
             new FabricTransportCallbackNotFoundException("FabricTransportCallbackNotFoundException"),
+            new FabricBackupNotFoundException("FabricBackupNotFoundException"),
+            new FabricSkipRestoreOperationException("FabricSkipRestoreOperationException"),
+            new FabricInsufficientMaxLoadCapacityException("FabricInsufficientMaxLoadCapacityException"),
```

Comment:
The testing approach looks like an afterthought. I doubt we have adequate coverage because I don't see the validation of the inner exceptions, error codes. We're comparing HResults but aren't setting them anywhere. We should put this under proper tests at some point.

Reply:
I agree, we should follow up on this.